tools/trap/ramdumpParser.py

```diff
@@ -820,6 +820,34 @@ def main():
 				ext = (line.split("=")[1].strip())
 	elf = (BIN_PATH + 'tinyara' + ext)
 
+	if not log_file and not dump_file:
+		print('Usage error: Must specify one of the -t or -e options. Plz find below for proper usage')
+		usage()
+
+	access_check_list = [
+		("gdb", gdb_path)
+		("nm", nm_path),
+		("readelf", readelf_path)
+	]
+
+	exist_check_list = [
+		("dump_file", dump_file),
+		("log_file", log_file),
+		("elf", elf)
+	] + access_check_list
+
+	for name, path in exist_check_list:
+		if path and not os.path.exists(path):
+			print(f"{path} does not exist. Please provide the proper path for {name}...")
```

Comment:
If we use F-strings, we need to mandate Python 3.6 or above in prerequisites

Reply:
Python 3.6 reached end-of-life.
And in Ubuntu 20.04 LTS, Python 3.8 is included.
How about mandate Python 3.8 or above in prerequisites?

Reply:
@sunghan-chang  Shall we go ahead with Python3.8 mandate ? OR
Shall we keep regular print statements across the tool ?

Reply:
In TizenRT docker, Python 3.7 is installed. So I think it is better to match with it.
How do you think @thapav  and @junmin-kim ?

Reply:
Agreed.
FYI) Python 3.7 will reach end-of-life in 2023.

Reply:
> In TizenRT docker, Python 3.7 is installed. So I think it is better to match with it. How do you think @thapav and @junmin-kim ?

Yes I think that would be better.
@junmin-kim Can you verify F-strings are working on 3.7

Reply:
I checked that f-string works at 3.7